test/test_front_matter_defaults.rb

```diff
@@ -27,6 +27,30 @@ class TestFrontMatterDefaults < JekyllUnitTest
     end
   end
 
+  context "A site with full front matter defaults (glob)" do
+    setup do
+      @site = fixture_site({
+        "defaults" => [{
+          "scope"  => {
+            "path" => "contacts/*.html",
```

Comment:
What happens if I have `*.html` here? Should it apply to `about.html` *and* `contacts/index.html`?